CHANGELOG.md

```diff
@@ -11,12 +11,16 @@ All notable changes to this project will be documented in this file.
 - Run a `containerdebug` process in the background of each Trino container to collect debugging information ([#687]).
 - Support configuring JVM arguments ([#677]).
 - Aggregate emitted Kubernetes events on the CustomResources ([#677]).
+- Support for Trino 470 ([#705]).
 
 ### Changed
 
 - Increased the default temporary secret lifetime for coordinators from 1 day to 15 days.
   This is because Trino currently does not offer a HA setup for them, a restart kills all running queries ([#694]).
 - Default to OCI for image metadata and product image selection ([#695]).
+- Explicitly set `fs.native-s3.enabled=true` and `fs.hadoop.enabled=true` in applicable catalog config properties ([#705]).
+  - Trino 470 requires the native S3 implementation to be used.
+- BREAKING: Always set the S3 region ([#705]).
```

Comment:
Can you please in here why this is breaking? Eg. "currently it does auto-detection(?), we will now default to us-east-1. Set region explicitly if using a different one"

Reply:
Updated in ab0526d6300d25549b07091230b5b24a8fdb8867